summit_blood_samples/blood_sample/models.py

```diff
@@ -234,10 +243,16 @@ def __str__(self):
 
     class Meta:
         indexes = [
-            models.Index(fields=['CreatedAt'], name='CreatedAt_pr_import_idx'),
+            models.Index(fields=['CreatedAt'],
+                         name='CreatedAt_pr_import_idx'),
         ]
 
 
+SITE_HELD = [
+    (0, 'UK Biocentre'),
```

Comment:
@johnmccabe44 is this the only option for Site Held? (I'm assuming so, but just sanity checking!)

Reply:
Yes, @jonc125